armi/reactor/tests/test_assemblies.py

```diff
@@ -704,7 +704,7 @@ def test_getBlockData(self):
             "residence": 3.145,
             "eqRegion": -1,
             "id": 299.0,
-            "bondRemoved": 33.7,
+            "bondRemoved": 0.337,
```

Comment:
This was changed because a setter was added to the `bondRemoved` and `gasReleaseFraction` parameters to enforce that these should be between `[0.0, 1.0]`. I believe anything less than 0.0 or greater than 1.0 is non-physical and should be an error.